doc/Language/syntax.pod6

```diff
@@ -654,16 +663,47 @@ See L<functions|/language/functions>.
     &f();  # Invoke &f, which contains a function
     &f.(); # Same as above, needed to make the following work
     my @functions = ({say 1}, {say 2}, {say 3});
-    @functions>>.();
+    @functions>>.(); # hyper method call operator
     =end code
 
+When declared within a class, a subroutine is named "method": methods
+are subroutines invoked against an object (i.e., a class instance).
+Within a method the special variable C<self> contains the object instance
+(see L<Methods|classtut#Methods>).
+
+
     =begin code :skip-test
     # Method invocation. Object (instance) is $person, method is set-name-age
     $person.set-name-age('jane', 98);  # Most common way
     $person.set-name-age: 'jane', 98;      # Precedence drop
     set-name-age($person: 'jane', 98);     # Invocant marker
     =end code
 
+<<<<<<< HEAD
+=======
```

Comment:
git conflict header?

Reply:
Shame on me, how did not catch them up (and I ran tests!)?
Removed.